xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -3825,13 +3828,12 @@ glusterd_compare_friend_volume(dict_t *peer_data, int32_t count,
     *status = GLUSTERD_VOL_COMP_SCS;
 
 out:
-    keylen = snprintf(key, sizeof(key), "%s.update", key_prefix);
-
     if (*status == GLUSTERD_VOL_COMP_UPDATE_REQ) {
-        ret = dict_set_int32n(peer_data, key, keylen, 1);
-    } else {
-        ret = dict_set_int32n(peer_data, key, keylen, 0);
+        /*Set the status to ensure volume is updated on the peer
+         */
+        arg->status_arr[(count / 64)] ^= 1UL << (count % 64);
```

Comment:
What's the significance of '64' here?

Reply:
The array type is unit64 so we need to use 64 to access a specific bit on a specific integer.